suite/runsuite_wrapper.pl

```diff
@@ -82,7 +82,7 @@
         $mydir = `/usr/bin/cygpath -wi \"$mydir\"`;
         chomp $mydir;
     }
-    system("ctest --output-on-failure -V -S \"${mydir}/runsuite.cmake${args}\" 2>&1");
+    system("ctest --output-on-failure -VV -S \"${mydir}/runsuite.cmake${args}\" 2>&1");
```

Comment:
Reminder to revert the extra V (as it pushes appveyor over the edge causing the web view to truncate and forcing devs to download the log just to see which tests failed which is far more common than needing the V)